src/docs/automation-and-resiliency/application-resource-tuning.md

```diff
@@ -0,0 +1,286 @@
+---
+title: Application Resource Tuning  
+
+slug: application-resource-tuning
+
+description: Describes how to tune resource limits and requests in OpenShift.
+
+keywords: Tuning, Resource, Limit, Request, Application OpenShift,
+
+page_purpose: Discusses application tuning, and provides resources and references for product teams.
+
+audience: developer
+
+author: Matt Spencer
+
+content_owner: 
+
+sort_order: 2
+---
+
+# Application Resource Tuning
+
+As touched upon in the [Resiliency Guidelines](/App-Resiliency-Guidelines#resourced), deploying applications with appropriate CPU and Memory [Requests and Limits](https://docs.openshift.com/container-platform/3.11/admin_guide/overcommit.html#requests-and-limits) is critical to ensure:
+
+* Resource availability for your own applications
+* Resource availability for other tenant applications
+
+While [Resource Quotas](./src/docs/automation-and-resiliency/openshift-project-resource-quotas) are quite generous, these quotas must be seen as a tool to allow tenants enough resources to temporarily burst usage for experimentation, rather than an upper limit of consistent use. The platform is not sized to support every tenant fully utilizing their _Resource Quota_.
```

Comment:
```suggestion
While [resource quotas](./src/docs/automation-and-resiliency/openshift-project-resource-quotas) are quite generous, these quotas must be seen as a tool to allow tenants enough resources to temporarily burst usage for experimentation, rather than an upper limit of consistent use. The platform is not sized to support every tenant fully utilizing their _Resource Quota_.
```

Reply:
You don't need to capitalize page names if it's part of a sentence. For example: Make sure you're using the right **resource quotas** [LINK] for your application. VS Review the **Resource Quotas page** [LINK] to make sure you're using the right resources for your application.